StoreKitUnitTests/TestHelpers/StoreKitTestHelpers.swift

```diff
@@ -57,6 +58,41 @@ extension StoreKitConfigTestCase {
 
 }
 
+extension StoreKitConfigTestCase {
+
+    /// Updates `SKTestSession.storefront` and waits for `Storefront.current` to reflect the change
+    /// This is necessary because the change is aynchronous within `StoreKit`, and otherwise code that depends
+    /// on the change might not see it in time, resulting in race conditions and flaky tests.
+    func changeStorefront(_ new: String) async {
```

Comment:
This method belonged to `StoreKitTests` class, but I moved it to `StoreKitConfigTestCase` because I needed it for one of my test case. Now, any test of `StoreKit` can change the Storefront.